posts/2017-01-10-dat-desktop.md

```diff
@@ -0,0 +1,41 @@
+# Dat Desktop Release
+
+Today we're announcing the release of Dat Desktop, a peer to peer data sharing
+app built for humans.
+
+Dat Desktop is a new way to share data with friends & colleagues. Using the
+peer to peer (p2p) Hyperdrive protocol, it makes sharing folders on your
+computer secure and convenient. No centralized servers needed.
```

Comment:
Description may still not convey what the app does. Definitely an image would be :1st_place_medal:. Maybe we could make this concrete (with gifs?!):

1. Drag a folder to the app
2. Send your Dat link to a friend
3. They will download your data